js/blocks/GraphicsBlocks.js

```diff
@@ -764,6 +764,11 @@ function setupGraphicsBlocks(activity) {
         }
 
         flow(args, logo, turtle, blk) {
+            
+            let newArgs = args[0]
+            if(args[0].toString().length>4){
```

Comment:
Since we have restricted the input previously, why not just limit the int value? This seems overly complicated.
Also, why 4 digits? And what about negative numbers?

Reply:
Sir, when I input numbers that are in 5 digits, my system starts lagging. Also, input greater than 50,000 also crashes chrome.
When I limit the digits to four, the minimum number that the user can input would be -999. 
Sir, I did not get the first statement. Which int value are we talking about?